go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/c9s/goprocinfo v0.0.0-20190309065803-0b2ad9ac246b
 	github.com/cilium/arping v1.0.1-0.20190728065459-c5eaf8d7a710
 	github.com/cilium/ebpf v0.0.0-20191113100448-d9fb101ca1fb
-	github.com/cilium/hubble v0.0.0-20200312104109-daf0d22074cb
+	github.com/cilium/hubble v0.0.0-20200320161706-5faac2a81350
```

Comment:
0.5.0 can be used now

Reply:
It wasn't tagged yet at the time of creating this PR. Anyway, this specific hubble version has the necessary changes, I'll bump to 0.5.0+ as part of the next PR.